stories/ChartTransformation.tsx

```diff
@@ -0,0 +1,622 @@
+// (C) 2007-2018 GoodData Corporation
```

Comment:
What about to rename this file with `Internal` prefix like AFM componens has? Like `InternalChartTransformation.tsx`